<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWin</class>
 <widget class="QWidget" name="mainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FindMinutiae</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="TabOptions">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="TabFileInspection">
      <attribute name="title">
       <string>File Inspector</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="ComboChooseFile"/>
       </item>
       <item>
        <widget class="Ui::ChartRecWidget" name="ChartShowRec" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="HorRecordControls">
         <item>
          <widget class="QPushButton" name="ButtonPreviousRecord">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../Resources/48x48/Prev.png</normaloff>../../Resources/48x48/Prev.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LabelRecordDescription">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Record Description</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonNextRecord">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../Resources/48x48/Successive.png</normaloff>../../Resources/48x48/Successive.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Ui::ChartRecSpecWidget" name="ChartShowRecSpectrum" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <zorder>ChartShowRec</zorder>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckLogScale">
         <property name="text">
          <string>Log scale</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabParameters">
      <attribute name="title">
       <string>Parameters</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="LabelParRecLength">
         <property name="text">
          <string>Record Length</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="LabelParGaussFilterRad">
         <property name="text">
          <string>Gaussian Filter Radius</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="SpinParGaussFilterRad">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="LabelParBackEstMinFilterRad">
         <property name="text">
          <string>Back. Est. Min. Filter Radius</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="LabelParBackEstMaxPeakWidthAllowed">
         <property name="text">
          <string>Est. Back. Max. Peak Width (relative)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="SpinParBackEstMaxPeakWidthAllowed">
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="LabelParBackEstDerEstimationDiam">
         <property name="text">
          <string>Est. Back. Boundary Deriv. Diam.</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="LabelParBackEstMaxIterations">
         <property name="text">
          <string>Est. Back. Max. Iterations</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="SpinParBackEstMaxIterations">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="LabelParBackEstMaxAllowedInconsistency">
         <property name="text">
          <string>Est. Back. Max. Inconsistency</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="LabelParBackEstMaxDistNodes">
         <property name="text">
          <string>Est. Back. Updating Radius</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="SpinParBackEstMaxDistNodes">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="LabelParIntervalStartFreq">
         <property name="text">
          <string>Starting Frequency</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QDoubleSpinBox" name="SpinParIntervalStartFreq">
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="value">
          <double>500.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="LabelParIntervalWidthFreq">
         <property name="text">
          <string>Freq. Interval Width</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QDoubleSpinBox" name="SpinParIntervalWidthFreq">
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="value">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="ComboParRecLength">
         <item>
          <property name="text">
           <string>16384</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18432</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20480</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="LabelParForeGaussFilterRad">
         <property name="text">
          <string>Foreground Gauss Filter Radius</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="SpinParBackEstMinFilterRad">
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="SpinParBackEstDerEstimationDiam">
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="SpinParBackEstMaxAllowedInconsistency">
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSpinBox" name="SpinParForeGaussFilterRad">
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="LabelParBinWidth">
         <property name="text">
          <string>Binning Matrix Bin Width</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QSpinBox" name="SpinParBinWidth">
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="TabIO">
      <attribute name="title">
       <string>Input/Output</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="HorInput">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="LabelInput">
           <property name="text">
            <string>Input Path or File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Ui::LineEditDropReceiver" name="LineEditInput"/>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonInputBrowse">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="HorOutput">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="LabelOutput">
           <property name="text">
            <string>Output Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Ui::LineEditDropReceiver" name="LineEditOutput"/>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonOutputBrowse">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="HorComputationControl">
         <item>
          <widget class="QPushButton" name="ButtonCompute">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Compute</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LabelCompletion">
           <property name="text">
            <string>Completion</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="indent">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Ui::ImageSlide" name="LabelMatrixView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="ButtonPrevImage">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LabelImageName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Image Name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonNextImage">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabDB">
      <attribute name="title">
       <string>DB Creation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="HorInputDB">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="LabelInputDB">
           <property name="text">
            <string>Input Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Ui::LineEditDropReceiver" name="LineEditInputDB"/>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonInputBrowseDB">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonCreateDatabase">
         <property name="text">
          <string>Create Database</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QChartView" name="BarSetDB" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonCleanPlot">
         <property name="text">
          <string>Clean Plot</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Ui::LineEditDropReceiver</class>
   <extends>QLineEdit</extends>
   <header location="global">Headers/LineEditDropReceiver.hpp</header>
  </customwidget>
  <customwidget>
   <class>Ui::ChartRecWidget</class>
   <extends>QWidget</extends>
   <header location="global">Headers/ChartRecWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Ui::ChartRecSpecWidget</class>
   <extends>QWidget</extends>
   <header location="global">Headers/ChartRecSpecWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Ui::ImageSlide</class>
   <extends>QLabel</extends>
   <header location="global">Headers/ImageSlide.hpp</header>
  </customwidget>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
